keyboard/src/lib/y2keyboard/keyboards.rb

```diff
@@ -334,6 +334,21 @@ def self.always_present_keyboards
     ]
   end
 
+  # @return [Array<String>]
+  def self.kbd_model_map_lines
+    filenames = [
+      "/usr/share/systemd/kbd-model-map",
+      # https://bugzilla.suse.com/show_bug.cgi?id=1211104
+      "/usr/share/systemd/kbd-model-map.xkb-generated"
+    ]
+    lines = filenames.map do |fn|
+      File.readlines(fn)
```

Comment:
IMHO exceptions should used only for *real* exceptions, if a file is optional I'd prefer using `File.exist` instead.

The code is then even simpler:
```ruby
File.exist(fn) ? File.readlines(fn) : []
```